akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowGraphCompileSpec.scala

```diff
@@ -178,109 +174,35 @@ class FlowGraphCompileSpec extends AkkaSpec {
       }.run()
     }
 
-    "attachSource and attachSink" in {
```

Comment:
I believe these removed tests are no longer relevant but correct me if you think otherwise.